service/src/main/kotlin/xyz/lebkuchenfm/api/eventstream/WebSocketEventStream.kt

```diff
@@ -6,31 +6,32 @@ import io.ktor.server.websocket.WebSocketServerSession
 import io.ktor.server.websocket.converter
 import xyz.lebkuchenfm.domain.eventstream.Event
 import xyz.lebkuchenfm.domain.eventstream.EventStream
-import java.util.Collections
-
-class Connection(val session: WebSocketServerSession)
+import xyz.lebkuchenfm.domain.eventstream.EventStreamConsumerId
+import java.util.UUID
```

Comment:
WDYT about using experimental kotlin uuid? (`@OptIn(ExperimentalUuidApi::class)`)

Reply:
I didn't want to use it because it's experimental, and was introduced only recently in Kotlin 2.0. I would prefer to wait for it to mature, and then swap it.